content/advocates/index/hassi-norlen.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Hassi Norlen
-image: hassi-norlen.png
+image: /images/advocates/hassi-norlen.png
```

Comment:
I'm curious about the updated path... 

is this to match Yaiza's updates?

Reply:
I didn't changed that. So, i supose yaiza knows it.
This PR has most commits from Yaiza an some css commits from me

Reply:
This makes sense...

Reply:
Yes, I did it. The previous advocate's card manually added the `/images/advocates/`, but when reusing the `AppCard`, I needed to add all the path to the mock data. Anyways, these .md files are going to disappear because we are going to recover all the data from airtable :)